ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -463,6 +464,66 @@ impl<'a> X64CodeGen<'a> {
         self.emit_call(inst_idx, func_decl_idx, &args)
     }
 
+    /// Codegen a (non-varargs) call.
+    fn cg_indirectcall(
+        &mut self,
+        inst_idx: InstIdx,
+        indirect_call_idx: &IndirectCallIdx,
+    ) -> Result<(), CompilationError> {
```

Comment:
Could any of `emit_call` be re-used (maybe with some adaption)?

Reply:
Yes. But I tried but gave up due to the different arguments each require.

Reply:
[d6e6c4f](https://github.com/ykjit/yk/pull/1141/commits/d6e6c4fff6090c26c9f785c5cc385de523d3f39d)

Reply:
The opposite. We might be able to funnel indirect calls through `emit_call`?

Reply:
Oh wait. i think I see what you mean.

Reply:
Yeah, I think remove the comment:
```
// FIXME: This call can probably be inlined, as it's not used anywhere else.
```

I think we will re-use emit_call at some point (maybe).

Reply:
I don't think we will tbh. But I'll remove the comment.

Reply:
[00197c1](https://github.com/ykjit/yk/pull/1141/commits/00197c13d0e66d0f23b944c62a237497d5cb25a7)